modules/alloy/src/main/ts/ephox/alloy/positioning/layout/Anchor.ts

```diff
@@ -14,7 +14,10 @@ export interface Anchor {
   origin: () => Origins.OriginAdt;
```

Comment:
Could the types here be `readonly`?

Reply:
Done.

Reply:
Arguably 99% of our object types, perhaps 100%, should have readonly fields